### DISCUSSION TITLE
Why kotlin code does not get generated?

### DISCUSSION THREAD

**Discussion by softshipper** [Netflix/dgs-framework#897] [Q&A] [answered]
Hi all,
The https://github.com/Netflix/dgs-codegen does not generate kotlin code as expected even I have set it explicitly to `language = "kotlin"` as you can see on the image below. 
<img width="381" alt="Screenshot 2022-02-27 at 19 13 06" src="https://user-images.githubusercontent.com/56756949/155894448-6a6723c4-b406-4dcb-af6e-c11c19cbcd82.png">
After executing the task `generateJava` the folder `dgs-codegen-generated-examples` is empty.  

Here is the repo.
[gql-dgs.zip](https://github.com/Netflix/dgs-framework/files/8149209/gql-dgs.zip)

Would be great if someone can help. 

Thanks

**Comment by srinivasankavitha** [comment, marked as answer]
It should generate all the kotlin classes under dgs-codegen. We do not generate examples for kotlin. We had initially implemented this for Java only and not kotlin. This feature is not actively maintained or widely used, so we will likely get rid of it sometime in future.

**Reply by willyxiao** [reply]
I also see that it's not generating the kotlin interfaces. Any idea why? I have a problem with implementing associated data right now with just data classes.

**Comment by srinivasankavitha** [comment]
The generation of example data fetchers is an experimental feature and not
supported in Kotlin. We plan to remove it altogether in the future for Java
as well.

**Comment by iamlmn** [comment]
Is this still the case? There is no explicit mention in the DGS documentation that generated-examples don't support Kotlin yet?

**Reply by srinivasankavitha** [reply]
Yes, this is not something we actively support and plan to drop soon for the Java case as well.
